src/cpu/flexcpu/inflight_inst.hh

```diff
@@ -204,17 +261,28 @@ class InflightInst : public ExecContext,
     }
 
     inline bool isCommitted() const
-    { return status() == Committed; }
+    { return status() >= Committed; }
```

Comment:
Need some comment about how the order now matters. Need comments both at the enum definition and here to say committed implies effAddred, etc.

Reply:
Added